src/assert.js

```diff
@@ -1,9 +1,14 @@
+import { pauseProcessing, resumeProcessing, objectValues } from "./core";
```

Comment:
Is the "./" always required here? I haven't seen that in ES6 module examples before.